hardhat.config.js

```diff
@@ -0,0 +1,51 @@
+require("@nomiclabs/hardhat-waffle");
```

Comment:
Not needed to be checked in, perhaps look at a `.gitignore` for this file if you want to keep it around locally